theme_common/static/src/scss/mixins.scss

```diff
@@ -264,7 +264,7 @@
 
 @mixin o-theme-cfa-header-shadows() {
     .bg-100, .bg-200, .bg-300, .bg-400, .bg-500, .bg-600, .bg-700, .bg-800, .bg-900 {
-        h1, h2, h3, h4, h5, h6 {
+        h1, h2, h3 {
             text-shadow: 0px 2px 0px $black;
         }
     }
```

Comment:
The effect is bad whatever the H size. It is only good when the text is white, for the darkest bg-XXX. Instead of disabling the effect for small H sizes, disable it for light bg-XXX

Reply:
Hello @qsm-odoo Thanks for the review!
I updated the fix :+1: 
(ps: good to know that bg-XXX also alter the text color)